<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Davide Bettio &lt;davide.bettio@kdemail.net&gt;</author>
 <class>ScreenSaver</class>
 <widget class="QWidget" name="ScreenSaver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>733</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="mSettingsGroup" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="mEnabledCheckBox">
          <property name="whatsThis">
           <string>Automatically start the screen saver after a period of inactivity.</string>
          </property>
          <property name="text">
           <string>Start a&amp;utomatically after:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KIntSpinBox" name="mWaitEdit"/>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>243</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="mLockCheckBox">
          <property name="whatsThis">
           <string>Prevent potential unauthorized use by requiring a password to stop the screen saver.</string>
          </property>
          <property name="text">
           <string>&amp;Require password after:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KIntSpinBox" name="mWaitLockEdit">
          <property name="whatsThis">
           <string>The amount of time, after the screen saver has started, to ask for the unlock password.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>243</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&amp;Screen locker type:</string>
        </property>
        <property name="buddy">
         <cstring>mSimpleLockerRadio</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="mSimpleLockerRadio">
        <property name="text">
         <string>S&amp;imple locker</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="mPlasmaWidgetsRadio">
          <property name="text">
           <string>&amp;Desktop Widgets</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KPushButton" name="mPlasmaSetup">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Configure...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="mScreenSaverRadio">
        <property name="text">
         <string>S&amp;creen saver</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QWidget" name="mScreenSaverSettings" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QTreeWidget" name="mSaverListView">
           <property name="whatsThis">
            <string>Select the screen saver to use.</string>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QWidget" name="mPreviewAreaWidget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>145</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QWidget" name="widget" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>5</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="KPushButton" name="mTestBt">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="whatsThis">
                <string>Show a full screen preview of the screen saver.</string>
               </property>
               <property name="text">
                <string>&amp;Test</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KPushButton" name="mSetupBt">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="whatsThis">
                <string>Configure the screen saver's options, if any.</string>
               </property>
               <property name="text">
                <string>&amp;Setup...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>5</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>145</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mEnabledCheckBox</tabstop>
  <tabstop>mWaitEdit</tabstop>
  <tabstop>mLockCheckBox</tabstop>
  <tabstop>mWaitLockEdit</tabstop>
  <tabstop>mSimpleLockerRadio</tabstop>
  <tabstop>mPlasmaWidgetsRadio</tabstop>
  <tabstop>mPlasmaSetup</tabstop>
  <tabstop>mScreenSaverRadio</tabstop>
  <tabstop>mSaverListView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mPlasmaWidgetsRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mPlasmaSetup</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mScreenSaverRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>mScreenSaverSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
